numpy/core/tests/test_multiarray.py

```diff
@@ -1382,6 +1382,21 @@ def test_pickle(self):
 
                 assert_equal(zs.dtype, zs2.dtype)
 
+    @pytest.mark.skipif(pickle.HIGHEST_PROTOCOL < 5,
+                        reason="requires pickle protocol 5")
+    def test_pickle_with_buffercallback(self):
+        array = np.arange(10)
+        buffers = []
+        bytes_string = pickle.dumps(array, buffer_callback=buffers.append,
+                                    protocol=5)
```

Comment:
@pitrou in the current draft of the PEP states that `buffer_callback` the example can accept a list of buffers with `buffer_callback=buffers.extend`. However, the implementation we test here only accepts a single buffer at a time. This is line with this section of the PEP: https://www.python.org/dev/peps/pep-0574/#id28

I think the example of the PEP should be fixed.

Reply:
Thanks! You're right, I forgot to fix this.